src/GDGUkraine/rest_controller.py

```diff
@@ -378,7 +378,10 @@ def send_confirm_participants(self, id, **kwargs):
                 secure_id = aes_encrypt(str(user_reg.id))
 
                 confirm_data = {
-                    'url': url_for('/confirm/{id}'.format(id=secure_id)),
+                    'url': url_for(
+                        'Controller.Root.confirm',
+                        type_='class-based',
+                        url_params={'aes_hash': secure_id}),
```

Comment:
`url_for` looks like generalization method, so I used it. Also,  what about bug I had, it was a param name. Method was looking for name of url_param `"aes_hash"`, but I've given it an `"id"`.

Maybe the `url_for` should throw more explicit error instead of naked `TypeError`